sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -703,6 +751,98 @@ def _to_outgoing_message(self):
+        try:
+            message_repr += ", application_properties={}".format(self.application_properties)
+        except:
+            message_repr += ", application_properties=<read-error>"
+        try:
+            message_repr += ", session_id={}".format(self.session_id)
+        except:
+            message_repr += ", session_id=<read-error>"
+        try:
+            message_repr += ", message_id={}".format(self.message_id)
+        except:
+            message_repr += ", message_id=<read-error>"
+        try:
+            message_repr += ", content_type={}".format(self.content_type)
+        except:
+            message_repr += ", content_type=<read-error>"
+        try:
+            message_repr += ", correlation_id={}".format(self.correlation_id)
+        except:
+            message_repr += ", correlation_id=<read-error>"
+        try:
+            message_repr += ", to={}".format(self.to)
+        except:
+            message_repr += ", to=<read-error>"
+        try:
+            message_repr += ", reply_to={}".format(self.reply_to)
+        except:
+            message_repr += ", reply_to=<read-error>"
+        try:
+            message_repr += ", reply_to_session_id={}".format(self.reply_to_session_id)
+        except:
+            message_repr += ", reply_to_session_id=<read-error>"
+        try:
+            message_repr += ", subject={}".format(self.subject)
+        except:
+            message_repr += ", subject=<read-error>"
+        try:
+            message_repr += ", partition_key={}".format(self.partition_key)
+        except:
+            message_repr += ", partition_key=<read-error>"
+        try:
+            message_repr += ", scheduled_enqueue_time_utc={}".format(self.scheduled_enqueue_time_utc)
+        except:
+            message_repr += ", scheduled_enqueue_time_utc=<read-error>"
+        try:
+            message_repr += ", auto_renew_error={}".format(self.auto_renew_error)
+        except:
+            message_repr += ", auto_renew_error=<read-error>"
+        try:
+            message_repr += ", dead_letter_error_description={}".format(self.dead_letter_error_description)
+        except:
+            message_repr += ", dead_letter_error_description=<read-error>"
+        try:
+            message_repr += ", dead_letter_reason={}".format(self.dead_letter_reason)
+        except:
+            message_repr += ", dead_letter_reason=<read-error>"
+        try:
+            message_repr += ", dead_letter_source={}".format(self.dead_letter_source)
+        except:
+            message_repr += ", dead_letter_source=<read-error>"
+        try:
+            message_repr += ", delivery_count={}".format(self.delivery_count)
+        except:
+            message_repr += ", delivery_count=<read-error>"
+        try:
+            message_repr += ", enqueued_sequence_number={}".format(self.enqueued_sequence_number)
+        except:
+            message_repr += ", enqueued_sequence_number=<read-error>"
+        try:
```

Comment:
I think `enqueued_time_utc` is also missing?

Reply:
added